plugins/mail/postfix_stats

```diff
@@ -3,50 +3,102 @@
 # Plugin to show Postfix statistics - needs pflogsumm
```

Comment:
If you like perfection, you may want to transform the current plugin documentation into the POD format (useful for `munindoc postfix_stats`). See the plugin "processes/multimemory" for an example.